rollup.config.js

```diff
@@ -9,12 +14,20 @@ export default [
       { file: pkg.main, format: 'cjs' }
     ],
     plugins: [
+      // Allows node_modules resolution
+      resolve({ extensions }),
+      // Compile TypeScript/JavaScript files
       babel({
+        extensions,
         babelrc: false,
         presets: [
           ['@babel/env', {
             modules: false
-          }]
+          }],
+          '@babel/typescript'
+        ],
+        plugins: [
+          '@babel/proposal-object-rest-spread'
```

Comment:
Do we need this in combination with TS?

Reply:
ah, probably not anymore. I was looking at rather old example